From 706717d8bbff85a4d4b7e08e809ce12a3e16bb42 Mon Sep 17 00:00:00 2001
From: Anish Trivedi <anish@freescale.com>
Date: Thu, 28 Apr 2011 11:56:52 -0500
Subject: [PATCH 220/442] ENGR00142728 Build test drivers when the
 peripheral's driver is loadable module

For the case where SCC and SAHARA drivers are compiled as loadable
modules, the respective test drivers are not built. Changed the Makefile
to build the test drivers even when the corresponding peripherals' drivers
are compiled as loadable module.

Signed-off-by: Anish Trivedi <anish@freescale.com>
---
 module_test/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/module_test/Makefile b/module_test/Makefile
index 4790b78..5f12d3b 100644
--- a/module_test/Makefile
+++ b/module_test/Makefile
@@ -42,7 +42,7 @@ sahara_test_driver-objs := km_test.o $(SAHARA_TEST_SOURCE)run_tests.o \
                            $(SAHARA_TEST_SOURCE)user_wrap.o
 
 
-obj-m += $(module_list-y)
+obj-m += $(module_list-y) $(module_list-m)
 
 EXTRA_CFLAGS := -DLINUX_KERNEL -DIMX_NEEDS_DEPRECATED_SYMBOLS
 
-- 
1.8.0

